packages/create-contentful-app/template/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
we can probably remove the favicon as this is going to be rendered in an iframe :)

Reply:
Yup, drop all meta tags but charset and viewport